src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,3 +237,9 @@ case class LibraryStatus(
     attachAllClusters: Boolean,
     statuses: List[LibraryClusterStatus])
 case class LibraryClusterStatus(clusterId: String, status: String)
+case class ContextId(id: String)
+case class ContextStatus(status: String, id: String)
```

Comment:
status should be an ADT
see https://github.com/ypg-data/sbt-databricks/pull/1/files#r30408327